packages/date-picker/src/DateRangePicker/DateRangePicker.spec.tsx

```diff
@@ -2,7 +2,8 @@ import React, { useEffect, useState } from "react"
 import { render, screen, waitFor } from "@testing-library/react"
 import userEvent from "@testing-library/user-event"
 import { DateRange } from "react-day-picker"
-import { DateRangePicker, DateRangePickerProps, formatDateRangeValue } from "./"
+import { formatDateRangeValue } from "./utils/formatDateRangeValue"
```

Comment:
How did this even happen 😅

Reply:
Tests failed if I didn't do it this way

Reply:
Yeah it sounds like the util used to be exported from the index, but no longer does. It's just weird that it wasn't caught earlier.

Reply:
It is exported from index, but oddly doesn't pick up:
```export * from "./DateRangePicker"
export * from "./utils"
```